config/checkstyle_checks.xml

```diff
@@ -350,6 +350,17 @@
     <module name="SuppressionXpathFilter">
       <property name="file" value="${checkstyle.suppressions-xpath.file}"/>
     </module>
+    <module name="SuppressionXpathSingleFilter">
+       <!--
+         Use suppressions.xml for suppressions, this is only example.
+         Properties will prevent filter from being valid.
+      -->
```

Comment:
this is not ok, we have use it for smth.
Such Filter will be awesome compromise to have all checkstyle settings in one file !!! Long awaited feature by users.

@rnveach , please review our discussions for last two new Checks that were created by @jodastephen , I remember that we mentioned that some cases are impossible to catch properly, so xpath filtering is required.

Reply:
Sir, sorry for that. I think it is just a mistake that I didn't correct in the documentation. It didn't use the suppressions.xml for suppressions. It is my fault.

Reply:
I have removed these lines of comments.

Reply:
done.